src/index.ts

```diff
@@ -786,6 +788,26 @@ const plugin: JupyterFrontEndPlugin<INotebookIntelligence> = {
       }
     });
 
+    app.commands.addCommand(CommandIDs.renameNotebookFromPython, {
+      execute: async args => {
+        const activeWidget = app.shell.currentWidget;
+        if (activeWidget instanceof NotebookPanel) {
+          const oldPath = activeWidget.context.path;
+          const newPath = args.newName as string;
```

Comment:
I think the new name should be input as just file name and it shouldn't change the parent directory of a notebook. example:
- oldPath: 'test/untitled.ipynb'
- newName: 'hello-world'
- newPath: 'test/hello-world.ipynb'